roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -125,13 +125,15 @@
 - name: Generate proxy session
   set_fact:
     session_secret: "{{ 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789' | random_word(200) }}"
+  when: session_secret is not defined
```

Comment:
I think if we wish to support the case you describe we need to treat as certs and write them to a file, do checks to see if said file exists.